packages/terra-show-hide/src/ShowHide.jsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Toggle from 'terra-toggle';
+import IconChevronDown from 'terra-icon/lib/icon/IconChevronDown';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './ShowHide.module.scss';
+import Button from './_ShowHideButton';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Content in the body of the component that will be shown or hidden
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Text that will be visible to the user while the component is collapsed
+   */
+  preview: PropTypes.string.isRequired,
+  /**
+   * Button text that will be displayed when the component is collapsed
+   */
+  collapsedText: PropTypes.string,
+  /**
+   * Button text that will be displayed when the component is expanded
+   */
+  expandedText: PropTypes.string,
```

Comment:
I'm confused about what expandedText and collapsedText actually represent.  Their names to me represent the body of text that will be displayed when the object is collapsed or expanded, however I think it's actually the clickable text of the object?  Would it be reasonable to instead call them something like "collapsedLinkText" or "collapsedButtonText"?

Reply:
Yeah I could see how that could be confusing to people. changed it in this commit:
https://github.com/gabeparra01/terra-core/pull/1/commits/cc61d5d48d64ecf13d09e4951cb2477e1a1f8a76